CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Upgrade `golang.org/x/sys/unix` from `v0.0.0-20210423185535-09eb48e85fd7` to `v0.0.0-20220919091848-fb04ddd9f9c8`.
   This addresses [GO-2022-0493](https://pkg.go.dev/vuln/GO-2022-0493). (#3235)
 
+### Fixed
+
+- Fix function `baggage.NewMember` to decode the `value` parameter instead of directly use it according to the W3C specification. (#3144)
```

Comment:
```suggestion
- Fix function `baggage.NewMember` to decode the `value` parameter instead of directly use it according to the W3C specification. (#3226 )
```

Reply:
fixed

Reply:
This does not appear resolved to me:

![20221013_080454](https://user-images.githubusercontent.com/5543599/195634375-969fb2b1-f5aa-4d2c-8c0b-79cdcee7c9ad.png)